src/components/ObitPane/ObitPane.css

```diff
@@ -0,0 +1,78 @@
+.obit {
+    background-color: rgb(62, 57, 57);
+    /* border: solid white 1px; */
+    display: grid;
+    grid-template-rows: 10% 35% 10% 35% 10%;
+    grid-template-columns: 10% 35% 10% 35% 10% ;
+    height: 87vh;
+    width: 99vw;
+}
+
+.triangle {
+    background-color: red;
```

Comment:
Do we still need this class? It looks like in the updated design there is no triangle class being used.

Reply:
Dont need it anymore! good catch